controllers/errors.controller.js

```diff
@@ -8,7 +8,7 @@ exports.handlePSQLErrors = (err, req, res, next) => {
   }
 
   if (err.code === "23502") {
-    res.status(400).send({ message: "Missing essential comment property." });
+    res.status(400).send({ message: "Incorrect data input." });
   }
   next(err);
```

Comment:
Once you've sent a response you shouldn't call `next(err)`. This will cause express to try to send multiple responses which can't happen, only the first will actually happen.